src/component/sheet.js

```diff
@@ -349,11 +349,11 @@ function cut() {
 }
 
 function paste(what, evt) {
-  const { data } = this;
+  const { data, datas } = this;
```

Comment:
datas is not a correct english word

Reply:
I'm well aware of that, couldn't be bothered to change the member names... will take care of it though